core/utils.py

```diff
@@ -35,17 +35,28 @@
 import urllib.request
 import zlib
 
-from core import feconf
 from core.constants import constants
 
+# The third party library path is defined in common.py,
+# which is required to import typing_extensions.
+from scripts import common # pylint: disable=unused-import
+
 from typing import (
     IO, Any, BinaryIO, Callable, Dict, Iterable, Iterator, List, Optional,
     TextIO, Tuple, TypeVar, Union, overload)
 from typing_extensions import Literal
 
+# Since feconf imports typing_extensions, it should be
+# imported after common is imported.
+from core import feconf # isort:skip  # pylint: disable=wrong-import-position
```

Comment:
Move this directly below the commons import.

Reply:
I had to move the common import below. The linter fails if I do it the other way round because of `isort:skip`.

Reply:
Done.